app/celery/process_ses_receipts_tasks.py

```diff
@@ -38,26 +38,23 @@ def process_ses_results(self, response):
             _check_and_queue_complaint_callback_task(*handle_complaint(ses_message))
             return True
 
-        aws_response_dict = get_aws_responses(ses_message)
-
-        notification_status = aws_response_dict["notification_status"]
         reference = ses_message["mail"]["messageId"]
 
         try:
             notification = notifications_dao.dao_get_notification_by_reference(reference)
         except NoResultFound:
             try:
                 current_app.logger.warning(
-                    f"RETRY {self.request.retries}: notification not found for SES reference {reference} (update to {notification_status}). "
+                    f"RETRY {self.request.retries}: notification not found for SES reference {reference}. "
```

Comment:
Arr losing these log info though... we could try a `get` instead with a default param value, to get an overridden None?

Reply:
yeah this annoyed me too. Figured making the smallest hot fix for now might be best, and followup with maybe thinking about how we might want to deal with the unparsed response type later.

Reply:
The exception is happening here
https://github.com/cds-snc/notification-api/blob/main/app/notifications/notifications_ses_callback.py#L39
basically if the response isn't Delivery", "Complaint", or "Bounce" then we don't know how to treat it and we raise an exception.

Reply:
Yes let's bring a follow up PR please to do this properly. 🙏